ros2nodl/test/ros2nodl/verbs/test_show.py

```diff
@@ -1,40 +1,47 @@
-# Copyright 2020 Canonical Ltd.
-#
-# This program is free software: you can redistribute it and/or modify it under the terms of the
-# GNU Limited General Public License version 3, as published by the Free Software Foundation.
-#
-# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
-# without even the implied warranties of MERCHANTABILITY, SATISFACTORY QUALITY, or
-# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Limited General Public License for more details.
-#
-# You should have received a copy of the GNU Limited General Public License along with
-# this program. If not, see <http://www.gnu.org/licenses/>.
+# Copyright (C) 2020 Canonical Ltd.
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License Version 3 as published by
+# the Free Software Foundation.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 import argparse
 from pathlib import Path
+from typing import List
 
+from ament_index_python import PackageNotFoundError
 import nodl
 import pytest
+from ros2nodl.verb import show
+
 
-from ros2nodl.verb.show import PackageNotFoundError, ShowVerb
+sv = show.ShowVerb()
```

Comment:
Maintaining state outside of individual tests is asking for trouble. Is there a reason this is required? pytest supports module-level fixtures if necessary, but this feels like something that could just be created within each test, too.

Reply:
fixed in 70d6ba5